src/e/component.spec.ts

```diff
@@ -5,9 +5,21 @@ describe('Engine', () => {
 
     beforeEach(() => {
         component = new Engine();
+        jest.spyOn(console, 'log').mockImplementation(() => null);
     });
 
-    it('should exist', () => {
-        expect(component).toBeTruthy();
+    it('should get stopped status at start', () => {
+        expect(component.status).toBe('stopped');
```

Comment:
Even though `.toBe` is probably fine in this scenario, would use `.toEqual` as default so that becomes the habit
```suggestion
        expect(component.status).toEqual('stopped');
```